### PR TITLE
feat(install/unix): add spicetify to session PATH

### PR DESCRIPTION
**Pull request by rxri**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -114,7 +114,9 @@ check() {
                 echo >> "$shellrc"
             fi
             echo "${2:-$path}" >> "$shellrc"
-            log "Restart your shell to have spicetify in your PATH."
+            # Source the rc file to have spicetify binary available already in the session
+            # shellcheck source=/dev/null
```
Comment (SunsetTechuila) [inline install.sh:118]:
```suggestion
            # shellcheck disable=SC1090
```

**Review by SunsetTechuila — DISMISSED** [review]

File: install.sh (hunk 1 above)
Comment (rxri) [reply install.sh:118]: the SC1090 mentions that the `shellcheck source=/dev/null` disables it

File: install.sh [hunk 2]
```diff
@@ -114,7 +114,9 @@ check() {
...
+            . "$shellrc"
```
Comment (SunsetTechuila) [inline install.sh:119]: this does work
```suggestion
            export PATH="$PATH:$spicetify_install"
```

**Review by SunsetTechuila — CHANGES_REQUESTED** [review]

File: install.sh (hunk 2 above)
Comment (theRealPadster) [reply install.sh:119]: I like this better ^

**Review by SunsetTechuila — APPROVED** [review]

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**